serverless.yml

```diff
@@ -86,6 +86,14 @@ functions:
           method: post
           cors: true
 
+  Generic:
+    handler: lambda_function.lambda_handler
+    name: socless_slack_generic
```

Comment:
Recommending `socless_slack_api_request` / `socless_slack_web_api_request` / something else that makes the purpose of this generic a bit clearer

Reply:
name has been changed